db/migrate/20240130154837_add_clinic_location_to_henry_ford.rb

```diff
@@ -0,0 +1,18 @@
+class AddClinicLocationToHenryFord < ActiveRecord::Migration[6.1]
+  def change
```

Comment:
It should be `up` and `down`

Reply:
I think that we do not need to use `up` and `down` because we don't need to revert created clinic locations.